requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+black==19.10b0
+faker
+pytest
+pytest-docker
+pyyaml
```

Comment:
Technically pyyaml is pulled in via some of these other packages it's not a terrible idea to list it anyway.